tests/integration/hive_metastore/test_catalog_schema.py

```diff
@@ -95,37 +150,35 @@ def test_create_catalog_schema_with_principal_acl_aws(
 
 @retried(on=[NotFound], timeout=timedelta(minutes=3))
 def test_create_catalog_schema_with_legacy_acls(
-    ws, make_user, make_catalog, make_schema, make_mounted_location, runtime_ctx, sql_backend
-):
-    src_schema = make_schema(catalog_name="hive_metastore")
-    src_external_table = runtime_ctx.make_table(
-        catalog_name=src_schema.catalog_name,
-        schema_name=src_schema.name,
-        external_csv=make_mounted_location,
-    )
-    dst_catalog = make_catalog()
```

Comment:
@FastLee : I removed this to test create a new catalog. Please review if the test is correct after my changes